package.json

```diff
@@ -33,7 +33,7 @@
     "jest-circus": "26.6.3",
     "jest-cli": "^26.0.1",
     "jest-styled-components": "^7.0.2",
-    "lerna": "^3.13.1",
+    "lerna": "^4.0.0",
```

Comment:
Lerna isn't a dependency of the project, only for the monorepo. There is also breaking changes here so this shouldn't be updated as it's just tooling for us to create releases

Reply:
Hi @derrickmehaffy sure. I can revert this change. 
Actually, the only breaking change is that Node 6.x and 8.x are no longer supported. Since Strapi v3 only supports Node >10.x, it should be a problem, right? https://github.com/lerna/lerna/releases/tag/v4.0.0

Reply:
We try not to touch tooling depends unless there is a reason to. As mentioned this doesn't affect strapi user's projects at all as it's only in the repo.

Reply:
Hey, just coming here to inform you that in terms of node version we're following the official node releases schedule & lifecycles. It means that we're soon going to deprecate Node 12 and add support for Node 18 (for this one, we still need some of our dependencies to allow it).

Reply:
```suggestion
"lerna": "^3.13.1",
```

Reply:
done!